python/cli/prelude_cli/views/interactive.py

```diff
@@ -873,7 +876,9 @@ def __init__(self, wiz: Wizard, title: str):
         self.title = title
 
     def enter(self):
-        webbrowser.open('https://platform.preludesecurity.com/detect', new=2)
+        d = dict(account=self.wiz.account.headers['account'], token=self.wiz.account.headers['token'])
+        data = base64.urlsafe_b64encode(json.dumps(d).encode('utf8')).decode('utf8')
+        webbrowser.open(f'https://platform.preludesecurity.com#{data}', new=2)
```

Comment:
we should use the ``hq`` entry from the keychain

Reply:
Unless we strip the `api.` part of the url

Reply:
Question: what if we instead hit the verify user route on Service (assuming we adjust it to always do a redirect, even on recertification)? Doing that could open the browser with the right account.. would that work?